xbmc/AppParamParser.cpp

```diff
@@ -35,6 +35,8 @@
 #endif
 #include <stdlib.h>
 
+using namespace KODI::MESSAGING;
```

Comment:
> was plenty of time for that

don't forget that we are short final for 15.0. So not plenty of time for someone who cares about this release. 

> A bit late to start arguing over namespace naming 

it is never too late to correct something that went wrong.